src/schema/v2/static_content.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  GraphQLObjectType,
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+
+const StaticContentType = new GraphQLObjectType<any, ResolverContext>({
+  name: "StaticContent",
+  fields: {
```

Comment:
You'll want to spread this in fields for the id, slug, internalID, etc. 

https://github.com/artsy/metaphysics/blob/da72816d9999383846ead5aa334676048d0ee6d3/src/schema/v2/artist/index.ts#L89

Reply:
You should actually probably add this to v1 too. That'll add the `__id` field which relay uses for caching.